src/components/LoginPage/LoginPageSagas.js

```diff
@@ -34,6 +33,6 @@ export function* handleAuth() {
 
 export function* loginInit() {
   yield googleApiLoader.load();
-  yield put(googleApiInitializedAction());
```

Comment:
Why this change?

Reply:
It's basically **console.log**, but made with redux and saga

Reply:
It's basically **console.log**

Reply:
It changes state
```
    case GOOGLE_API_INITIALIZED_ACTION:
      return Object.assign({}, state, { isGoogleApiInitialized: true });
```
so we can then know if `gapi` is already loaded or not. If it's not loaded we can't authorize the user: the user will click "login" and the app will break.

Reply:
We don't need it because auth code is run on `REQUEST_AUTHORIZATION_ACTION`, but `REQUEST_AUTHORIZATION_ACTION` only dispatched when `isGoogleApiInitialized: true`.

So basically we have a state field to change a click handler. Click handlers is safe to assign because they are not immediately executed.

Reply:
fixed by https://github.com/DarthVanger/NeuralNotes/pull/167/commits/2cf00e0e0593bd1f37ac43153229f3446b782603